features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/wso2/templates/stats/api-usage-destination/js/stats.js

```diff
@@ -141,7 +141,6 @@ var drawAPIUsageByDestination = function(from,to){
                              $('#destinationBasedUsageTable_paginate').show();
                      },
                     });
-                    $('select').css('width','60px');
```

Comment:
why is this removed?

Reply:
This overrides the default wso2 styling and causes API Provider drop-down list to shrink.